File: tests/human/x/python/bool_chain.py
def boom():
    print("boom")
    return True

print((1 < 2) and (2 < 3) and (3 < 4))
print((1 < 2) and (2 > 3) and boom())
print((1 < 2) and (2 < 3) and (3 > 4) and boom())

File: tests/human/x/rb/bool_chain.rb
def boom
  puts 'boom'
  true
end

puts ((1 < 2) && (2 < 3) && (3 < 4)).to_s
puts ((1 < 2) && (2 > 3) && boom).to_s
puts ((1 < 2) && (2 < 3) && (3 > 4) && boom).to_s
